plugins/modules/clustering/nomad/nomad_job_info.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2020, FERREIRA Christophe <christophe.ferreira@cnaf.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: nomad_job_info
+author: FERREIRA Christophe (@chris93111)
+version_added: "1.2.0"
+short_description: Get Nomad Jobs info
+description:
+    - Get info for one Nomad job.
+    - List Nomad jobs.
+requirements:
+  - python-nomad
+options:
+    host:
+      description:
```

Comment:
I think it would make sense to move all the common options from both modules to a doc_fragment. See for example https://github.com/ansible-collections/community.general/blob/main/plugins/doc_fragments/hetzner.py, and https://github.com/ansible-collections/community.general/blob/main/plugins/modules/net_tools/hetzner_failover_ip.py#L25-L26 on how to use it.